src/prelude.rs

```diff
@@ -19,6 +19,7 @@ pub use bevy::ui::FocusPolicy;
 pub use bevy::ui::Val::*;
 pub use bevy_asset_loader::prelude::*;
 pub use bevy_kira_audio::prelude::*;
+pub use bevy_simple_prefs::{Prefs, PrefsPlugin};
```

Comment:
```suggestion
pub use bevy_simple_prefs::Prefs;
pub use bevy_simple_prefs::PrefsPlugin;
```
Alternatively, this could be omitted from the prelude if we intend to have only 1 global preferences struct, which I think is reasonable for a jam template.